addok/helpers/scripts.py

```diff
@@ -0,0 +1,13 @@
+from pathlib import Path
+
+from addok.config import config
+from addok.db import DB
+
+
+@config.on_load
+def load_scripts():
+    root = Path(__file__).parent / 'lua'
+    for path in root.glob('*.lua'):
+        with path.open() as f:
+            name = path.name[:-4]
+            globals()[name] = DB.register_script(f.read())
```

Comment:
Ahah, I knew that dealing with `globals` will create me troubles :p
Those scripts are commited into the core, there is not risk that a user create such a script by mistake and commit and does not see that the tests are broken and release a new version and… :)
I wanted a simple API like "scripts.xxxx", and we need to wait for the config to be loaded for the DB to be instanciated, to be able to register the scripts.